terraform/modules/hub/egress_proxy.tf

```diff
@@ -28,6 +28,18 @@ resource "aws_security_group" "egress_via_proxy" {
   vpc_id = aws_vpc.hub.id
 }
 
+resource "aws_security_group_rule" "egress_to_proxy_task" {
```

Comment:
I'd find it less confusing if this rule was next to the equivalent `ingress` rule below

Reply:
Can I move them around once I've deleted all the old stuff?

Reply:
sure